packages/grafana-prometheus/src/components/metrics-browser/MetricSelector.tsx

```diff
@@ -31,7 +31,7 @@ export function MetricSelector() {
             data-testid={selectors.components.DataSource.Prometheus.queryEditor.code.metricsBrowser.selectMetric}
           />
         </div>
-        <Label description="Set to 'none' to remove limit and show all labels for a selected metric. Removing the limit may cause performance issues.">
+        <Label description="Set to 0 (zero) to remove limit and show all labels for a selected metric. Removing the limit may cause performance issues.">
```

Comment:
```suggestion
        <Label description="Remove limit or set it to 0 to display all metrics, labels, and values. Removing the limit may cause performance issues.">
```

Reply:
I think it make sense to make it here clear that it affects all - metrics labels and values. Also users can remove limit it or set it to 0 to get all results, right?

Reply:
Yeah let's make it clear. 
Setting it zero means prometheus will return everything. If user removes the number from the field (input is an empty string in this case) we fallback to the default limlt. I'll make sure this is clear.

Reply:
https://github.com/grafana/grafana/pull/104101/commits/f18fc84466c4cc6a81fcbb5c763c31a5325a0d72

Reply:
Nice! You made it even clearer. 👏